{
	"candidate": "CVE-2019-16170",
	"description": "An issue was discovered in GitLab Enterprise Edition 11.x and 12.x before 12.0.9, 12.1.x before 12.1.9, and 12.2.x before 12.2.5. It has Incorrect Access Control.",
	"notes": {
		"msalvatore": "Affects GitLab EE 11.6 and later."
	},
	"priority": "medium",
	"cvss": {
		"nvd": " CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:L/A:N [7.1 HIGH]"
	},
	"bugs": [
		"http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=940007"
	],
	"references": [
		"https://about.gitlab.com/2019/09/10/critical-security-release-gitlab-12-dot-2-dot-5-released/",
		"https://www.cve.org/CVERecord?id=CVE-2019-16170"
	],
	"public_date": "2019-09-16 12:15:00 UTC",
	"packages": {
		"bionic": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"devel": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"disco": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"eoan": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"esm-apps/xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "not-affected",
				"note": "code not present"
			}
		},
		"esm-infra-legacy/trusty": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"precise/esm": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"trusty": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"trusty/esm": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"upstream": {
			"gitlab": {
				"name": "gitlab",
				"status": "needs-triage"
			}
		},
		"xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "not-affected",
				"note": "code not present"
			}
		}
	}
}
